src/main/java/h13/controller/scene/SceneSwitcher.java

```diff
@@ -29,7 +29,7 @@ public enum SceneType {
          */
         MAIN_MENU(() -> SceneAndController.fromScene(new MainMenuScene())),
         /**
-         * The about scene.
+         * The "about" scene.
```

Comment:
Hmm entweder bei allen dann Anführungszeichen oder mit Bindestrich würd ich sagen, das bitte anpassen

Reply:
Addressed in the new commit, among some other changes